### DISCUSSION TITLE
barcodes and update

### DISCUSSION THREAD

**Discussion by BorampetBhavani** [inventree/InvenTree#7307] [General]
Hello I am new to this platform and I am a recent graduate student . when i am working on inventree , when i scan any barcode its not giving any result. app version is 0.13.3 and also how do i need to update the version please help me out.

**Comment by SchrodingersGat** [comment]
@BorampetBhavani you'll need to provide some more data:

- What sort of barcode you are trying to scan
- Please elaborate on "not giving any result". Is it giving a "fail" notification and potentially a message?

**Comment by matmair** [comment]
I would appreciate if you would not post the same text multiple times in different discussions , this creates unnecessary triaging effort for maintainers
https://github.com/inventree/InvenTree/discussions/5689#discussioncomment-9530043
